src/emc/rs274ngc/interp_cycles.cc

```diff
@@ -154,7 +154,7 @@ int Interp::convert_cycle_g83(block_pointer block,
      Thanks to Billy Singleton for pointing it out... */
   CHKS((delta <= 0.0), NCE_NEGATIVE_OR_ZERO_Q_VALUE_USED);
 
-  rapid_delta = G83_RAPID_DELTA;
+  rapid_delta = _setup.parameter_drill_cycle_chip_break_distance;
```

Comment:
Why not change the definition of G83_RAPID_DELTA instead?  If G83_RAPID_DELTA is no longer used, it should be removed from src/emc/rs274ngc/interp_internal.hh.  I suspect it is better to keep using it, but changing its definition to _setup.parameter_drill_cycle_chip_break_distance.

Reply:
mostly because the G83_RAPID_DELTA variable is a constant.
I think it is also more informative to see where the value came from. (_setop)

yes G83_RAPID_DELTA should be removed.